### ISSUE TITLE
Labels Are Not Accessible

### ISSUE THREAD

**Issue by philippfromme** [bpmn-io/form-js#32]
__Describe the Bug__

HTML inputs are not connected to their labels. The `for` and `name` are missing and therefore accessibility is not given.

```html
<label class="fjs-form-field-label">Invoice Number</label>
<input class="fjs-input" type="text" disabled="">
```

__Expected Behavior__

Expected HTML:

```html
<label for="foo" class="fjs-form-field-label">Invoice Number</label>
<input name="foo" class="fjs-input" type="text" disabled="">
```

__Environment__

- Library version: v0.0.8
